index.html

```diff
@@ -0,0 +1,198 @@
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <p>"The software of your thoughts"</p>
+          <h1>WE COME WITH SMART TECHNO PRODUCTS!</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            In simple words, technology is when we take the concepts of science
+            and transform them into gadgets and devices that are useful to us as
+            human beings. When we look around us, almost everything is a result
+            of or a form of technology.
+          </p>
+        </div>
+        <div class="address-txt">
+          <p><strong>Next Zoom conference(monday july 25/2022)</strong></p>
+          <p>@ BahirDar, Ethiopia</p>
+        </div>
+      </section>
+
+      <!-- Main Program -->
+
+      <section class="main-program">
+        <h2 class="main-pro">See The All Program</h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Coding</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Language</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-mobile fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Computer</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Programming</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
```

Comment:
- [ ] Kindly ensure that in the mobile version the cards are layout horizontally, with the image at the left, title in the center, and description at the right, instead of vertically  as you can see below
> Your Design
![image](https://user-images.githubusercontent.com/90556221/181480901-54ecb2da-eabb-46e1-8312-a1cb0a045d98.png)

> Behance Design
![image](https://user-images.githubusercontent.com/90556221/181481090-d123df33-4a80-44ab-a346-dda8bb186222.png)